test/existing_tag/libbaz/wscript

```diff
@@ -4,15 +4,19 @@
 APPNAME = 'libbaz'
 VERSION = '1.0.0'
 
+
 def resolve(ctx):
     pass
 
+
```

Comment:
I think it is totally useless to have 2 empty lines between top-level function. I see that you had to add spaces at million spaces. Let's not waste any effort on that in the future. A single separating line is perfectly fine.